### DISCUSSION TITLE
Custom Naming convention post deployment?

### DISCUSSION THREAD

**Discussion by thefeds-nau** [Azure/Enterprise-Scale#412] [Q&A] [answered]
Hello, Been reading up on this deployment and have a question on the naming scheme. In the image for Enterprise-Scale it shows the name AdventureWorks under Tenant Root Group. And then I see the advworks- prefix throughout. I see that I can add my own prefix when i clicked the deploy to Azure, but I am wondering if there will be an AdventureWorks Management Group under my Root Tenant after deployment? We are a brownfield Tenant, and I would like to begin migrating our current subscriptions into this model, but do not want to have the AdventureWorks Management Group. Thank you
Fed

**Comment by krnese** [comment, marked as answer]
Hey - thanks for reaching out! The prefix you provide during deployment will be present as part of the management groups that will be created, as well as the platform resources you deploy (hub, log analytics workspace etc.). I.e., you will not end up with 'Contoso' or 'Adventure Works' prefix in your tenant.

Regarding the existing environment and how to transition into Enterprise-Scale architecture, pls see if this article helps: https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/enterprise-scale/transition

**Comment by pinakighatak** [comment]
Is there a possibility of using your own naming convention to start with? Currently there is only a prefix possibility. I have customers who want something like this:
MG- (not underscore) for management group
AZPOL- for policies
RG-<Environment>-<function> for resource groups

**Reply by jtracey93** [reply]
Hey @pinakighatak, Please raise this as a feature request via the issues blade 👍

**Reply by bradcart1971** [reply]
Hi folks. I love the work you've done here.

I'm deploying the Enterprise (small) Landing Zone on the 6th.  My customer wants to customize all of the follow naming conventions to remain consistent with the on-prem naming convention. We are deploying the Enterprise (small) in production next week. 

I don't see any other posts on custom naming within this forum. Has this feature been integrated in the Accelerator yet? Or is there a way for me to export all raw files to GitHub and modify the raw files? 

They want to customize all of the follow naming conventions:
Information Tech			
Operations Tech				
Domain services	
Monitoring and Management	
Backup Services	
Security Mgmt Services	
Firewall Enforcement Pnt	
Development	
Production	
Quality Assurance Testing	
Business Continutiy 	
Environment

**Reply by bradcart1971** [reply]
@pinakighatak ?
